libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/MapboxReplayer.kt

```diff
@@ -128,6 +131,7 @@ class MapboxReplayer {
      * When initializing or testing an app, it is needed and useful to push a device location.
      * This helper function can be used to push the actual location into the replayer.
      */
+    @RequiresPermission(anyOf = [ACCESS_COARSE_LOCATION, ACCESS_FINE_LOCATION])
```

Comment:
The Kotlin compiler doesn't fail a build because of the `RequiresPermission` annotation. But Android Lint does. I'm wondering, is this a breaking change? Probably no. 🤔 
@mapbox/navigation-android , what do you think?

Reply:
I don't think it's breaking changes if you can still build an app with `./gradlew build`